README.md

```diff
@@ -3,40 +3,41 @@
+[Scott Weber](https://adhocteam.us/our-team/scott-weber/) and [John French](https://adhocteam.us/our-team/john-french/) set out to explore the potential value of adding animation and motion directives into the [CMS Design System (CMSDS)](https://design.cms.gov/). Currently, there is no framework or guidance for animation within the CMSDS. **We spent two days figuring out if that would be valuable and how it could be successful.**
+{:.ds-text--lead}
 
 ## Why explore adding animation?
 
 When used correctly, motion and animation can broaden the toolkit designers and developers have to help enrich digital services. It can help give a user confirmation that a task has been completed, or let them know that things are happening in the background.
 
 Animation can be used as a part of a larger strategy to improve inclusivity and accessibility of services by adding another layer of context to digital interactions. It can help to highlight connections between elements and information in real time or provide simulated haptic feedback when a button is pressed or selected.
 
-For our work at the Centers for Medicare and Medicaid, defining a standardized framework for animation in line with the CMS Design System will allow dozens of teams and projects to use contribute to improving animations standards together across the agency. This will lead to consistent, reliable implementation of animation across CMS digital services.
+For our work at the Centers for Medicare and Medicaid, defining a standardized framework for animation in line with the CMSDS will allow dozens of teams and projects to use and contribute to the animation standards together. This will lead to consistent, reliable implementation of animation across CMS digital services.
 
-## Defining the interactions
+## Defining motion and animation options
 
-We had to start by assessing common UI animation methods that would provide the highest flexibility and value across all components that already exist in the CMS Design System. The idea was essentially to create “animation classes” that can be added to any element in the same way the design system handles CSS utilities now.
+We started by assessing common UI animation methods that would provide the highest flexibility and value in the CMSDS. The idea was essentially to create “animation classes” that can be added to an element in the same way the design system handles CSS utilities now.
+
+We came up with the following list: 
 
 * Fade In
-* Fade In + Expand
+* Fade In + Grow
 * Fade Out
-* Fade Out + Subtract
+* Fade Out + Shrink
 * Color
-* Jiggle
-* Shake
-* Expand
-* Subtract
-* Subtract + Fade Out
-* Slide
+* Shake (Directional variations)
+* Grow
+* Shrink
+* Shrink + Fade Out
```

Comment:
Changed "Expand" to "Grow" and "Subtract" to "Shrink" I feel that is more descriptive. Also removed "Jiggle" in favor of having "Shake" offer various directions.